src/Tizen.NUI/src/public/Window/BorderWindow.cs

```diff
@@ -137,10 +137,34 @@ internal void UpdateProperty()
         {
             if (borderInterface != null)
             {
+                if (borderLineThickness != borderInterface.BorderLineThickness)
+                {
+                    int diffBorderLine = (int)(borderInterface.BorderLineThickness - borderLineThickness);
```

Comment:
currently, `borderLineThickness` is `uint` so it's not a big deal..but anyway,

Could we cast unsigned-to-signed value by value, for safety? if `borderLineThickness` type changed, it will make int overflow.

```suggestion
                    int diffBorderLine = (int)borderInterface.BorderLineThickness - (in t)borderLineThickness;
```

Reply:
수정했습니다.~